prometheus/prometheus.yml

```
@@ -13,6 +13,18 @@ scrape_configs:
     static_configs:
       - targets: ['host.docker.internal:19006']
 
+  - job_name: 'chatbot-service'
+    metrics_path: '/actuator/prometheus'
+    scrape_interval: 5s
+    static_configs:
+      - targets: [ 'host.docker.internal:19012' ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)